exporters/dsexporter/README.md

```diff
@@ -0,0 +1,17 @@
+# dsexporter
+
+## Overview
+
+The `dsexporter` is a specialized Prometheus exporter designed to provide availability metrics for the components
+within the Konflux ecosystem. It generates a custom metric (`request_count`) related to the number of requests handled
```

Comment:
request_count is not relevant anymore since we are updating the code with actual metrics.